lib/sisimai/bite/email/domino.rb

```diff
@@ -0,0 +1,174 @@
+module Sisimai::Bite::Email
+  module Domino
+    # Sisimai::Bite::Email::Domino parses a bounce email which created by IBM
+    # Domino Server. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Domino.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :subject => %r/\ADELIVERY FAILURE:/,
+      }
+      Re1 = {
+        :begin   => %r/\AYour message/,
+        :rfc822  => %r|\AContent-Type: message/delivery-status\z|,
+        :endof   => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReFailure = {
+        userunknown: %r{(?>
+           not[ ]listed[ ]in[ ](?:
+             Domino[ ]Directory
+            |public[ ]Name[ ][&][ ]Address[ ]Book
+            )
+          |Domino[ ]ディレクトリには見つかりません
+          )
+        }x,
+        filtered:    %r/Cannot[ ]route[ ]mail[ ]to[ ]user/x,
+        systemerror: %r/Several[ ]matches[ ]found[ ]in[ ]Domino[ ]Directory/x,
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'IBM Domino Server'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
+      def headerlist;  return []; end
+      def pattern;     return Re0; end
+
+      # Parse bounce messages from IBM Domino Server
```

Comment:
__[rubocop]__
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->